src/inc/regdisp.h

```diff
@@ -8,7 +8,7 @@
 
 #ifdef DEBUG_REGDISPLAY
 class Thread;
-
+struct REGDISPLAY;
```

Comment:
Why was this change not required for x64 in that case? Is the structure defined in some manner for x64 differently than for arm64?